src/layout/header/Header.tsx

```diff
@@ -100,12 +100,6 @@ export const Header = () => {
             display: 'flex',
             alignItems: 'center',
             gap: '1rem',
-            'a, button': {
-              flexDirection: 'column',
-              '.MuiButton-startIcon': {
-                margin: 0,
-              },
-            },
```

Comment:
Dette var gjort slik for å unngå duplisering, men når vi nå har ett element her som skal være annerledes  (logg-inn-knappen) så går det litt i ball. Smerter å duplisere dette flere plasser nå, men tror det likevel er bedre slik for nå. Tar gjerne i mot forslag til en bedre løsning da